### DISCUSSION TITLE
"the camera is not responding" on remote access

### DISCUSSION THREAD

**Discussion by bachjessen** [koush/scrypted#1138] [Q&A]
**Describe the bug**
When i'm connected to my wifi (same as ring and tapo c100) I I'm able to stream but when I'm remote on 5g or other wifi

**To Reproduce**
1. Go to settings and turn off wifi
2. Click on your homekit camera and I get a "the camera is not responding"
3. reanble wifi and connect, the cameras respond

**Expected behavior**
responding cameras on 5g and other wifi's worked before

**Screenshots**
![7776195D-75C3-41FF-8B28-AADEB22EB056_1_101_o](https://github.com/koush/scrypted/assets/59052984/65b7e54c-d1b6-4c0a-a399-ef6d58d1735e)

**Smartphone (please complete the following information):**
 - Device: 13promax
 - OS: iOS 17.0.3
 - Browser ARC
 - Version 0.57.0

**Additional context**

**Comment by joshua-ballard** [comment]
I'm actually having this issue as well, but with only 1 of my 5 cameras.  I have 4 C110 and 1 C100.  At home connected to wifi, all work.  When I'm remote (via VPN or not), the C100 shows not available.
